pulp_2to3_migration/app/plugin/deb/migrator.py

```diff
@@ -0,0 +1,50 @@
+import asyncio
+
+from pulp_2to3_migration.app.plugin.api import (
+    Pulp2to3PluginMigrator,
+    ContentMigrationFirstStage,
+    DeclarativeContentMigration,
+)
+
+from pulp_deb.app import models as pulp3_models
+
+from . import pulp2_models
+from . import pulp_2to3_models
+from .repository import DebImporter, DebDistributor
+
+
+class DebMigrator(Pulp2to3PluginMigrator):
+    """
+    An entry point for migration the Pulp 2 Debian plugin to Pulp 3.
+
+    Attributes:
+      Check parent class.
+    """
+    pulp2_plugin = 'deb'
+    pulp2_content_models = {
+        'deb': pulp2_models.DebPackage,
+    }
+    pulp2_collection = 'units_deb'
+    pulp3_plugin = 'pulp_deb'
+    pulp3_repository = pulp3_models.DebRepository
+    content_models = {
+        'deb': pulp_2to3_models.Pulp2DebPackage,
+    }
+    importer_migrators = {
+        'deb_importer': DebImporter,
+    }
+    distributor_migrators = {
+        'deb_distributor': DebDistributor,
+    }
+
+    @classmethod
+    def migrate_content_to_pulp3(cls):
+        """
+        Migrate pre-migrated Pulp 2 Debian content.
+        """
+        first_stage = ContentMigrationFirstStage(cls)
+        dm = DeclarativeContentMigration(first_stage=first_stage)
```

Comment:
So so far, I am simply trying to use the default migration pipeline. (The same one that `pulp_file`/`iso` uses.)
I am not sure if this naive approach could possibly work, since I am not that well versed in the topic of pulp3 async pipelines.
Any pointers are appreciated.

Reply:
It can work, it depends on your content.
How simple is your sync workflow: do you have a custom sync pipeline? Does order of content imported matters? Do you need to build any relations between different content units after they are in Pulp?

Reply:
I looked closer at the code you are adding. Since for now it's just a debian package migration. I think it should work with default first stage and pipeline.

Reply:
My plan/hope is that I can get only packages to work in a first step (keeping it as simple as possible), and then look into "Releases" and "Components" in a second step (at which point I will most likely need to adjust the pipeline).
Repositories containing only packages have an independent use case in pulp_deb so I think this approach makes sense.